### ISSUE TITLE
assigning compute address to nat_ip causing crash.

### ISSUE THREAD

**Issue by mikesiegel** [terraform-providers/terraform-provider-google#604]
### Terraform Version
Terraform v0.10.7

### Affected Resource(s)

- google_compute_instance
- google_compute_address

### Terraform Configuration Files
```
resource "google_compute_address" "exampleaddress" {
  name = "example-address"
}

resource "google_compute_instance" "example" {
  count = 1

  name         = "tf-example-${count.index}"
  machine_type = "n1-standard-1"
  zone         = "${var.region_zone}"
  tags         = ["example-node"]
  depends_on   = ["google_compute_address.exampleaddress"]

  network_interface {
    network = "default"

    access_config {
      nat_ip = "${google_compute_address.exampleaddress.0.address}"
    }
  }
...
}

```

### Panic Output

https://gist.github.com/mikesiegel/0f193a6b5e8b20a8c5a85bbb0bb31a3a

### Expected Behavior
IP address created, applied to a single instance..

### Actual Behavior
Crash.

### Steps to Reproduce

1. `terraform apply`

Note that this is being used elsewhere in the config, is populated with a value and works fine (in google_sql_database_instance authorized_networks for example). Commenting out the single line (and taking an ephemeral IP) produces expected results.

**Comment by emilymye** [comment]
There is also a newer version of the provider v1.1.0 so you might want to try using that if possible. However, I can't seem to reproduce this with the given config for either v1.1.0 or v.1.0.1. Is there any other config you are using?

**Comment by rosbo** [comment]
The crash was fixed in #556. 

Feel free to reopen this issue if you are still having trouble after updating the terraform google provider to the latest version.
